src/OneExplorer/OneExplorer.ts

```diff
@@ -261,8 +261,7 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
         .then(ans => {
           if (ans === approval) {
             Logger.info('OneExplorer', `Delete '${oneNode.node.name}'.`);
-            return vscode.workspace.fs.delete(
-                oneNode.node.uri, {recursive: recursive, useTrash: true});
+            vscode.workspace.fs.delete(oneNode.node.uri, {recursive: recursive, useTrash: true});
```

Comment:
Please do not remove `return` here as it must return a promise to let the next then phrase know that the promise was done.